### PR TITLE
merge Sen's chatgpt branch in master

### PR DESCRIPTION
**Pull request by monperrus**

### CODE REVIEW

**Comment by monperrus** [comment]
@TomasAndersonFang what do you think? could we merge that into master?

File: src/core/benchmarks/Defects4J.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import subprocess
 # from sets import Set
 
-from config import DATA_PATH, JAVA8_HOME, JAVA8_HOME, TEST_TIMEOUT
```
Comment (monperrus) [inline src/core/benchmarks/Defects4J.py:8]: JAVA8 seems more recent. WDYT?

**Comment by TomasAndersonFang** [comment]
Hi Martin,

I will check and clean my code again and then inform Pengyu to merge.

Best,
Sen

**Comment by monperrus** [comment]
great!

**Comment by TomasAndersonFang** [comment]
@zpengyu Hi Pengyu, I checked and cleaned my code, and I think you can merge it. If you have any questions, please don't hesitate to contact me.

**Comment by zpengyu** [comment]
Code looks good to me. Usually branch owner should click the "merge" button as long as the PR got approved.

So you can merge it by yourself :) @TomasAndersonFang

**Comment by monperrus** [comment]
great to see that important branch merged!

**Pull request merged**